README.md

```diff
@@ -13,52 +13,23 @@
 
 This module creates a secure landing zone within a single region.
 
----
-
-## Table of Contents
-
-1. [VPC](#vpc)
-    - [VPCs Variable](#vpcs-variable)
-    - [Flow Logs](#flow-logs)
-3. [Transit Gateway](#transit-gateway)
-4. [Security Groups](#security-groups)
-    - [Security Groups Variable](#security-groups-variable)
-5. [Virtual Servers](#virtual-servers)
-    - [VPC SSH Keys](#vpc-ssh-keys)
-    - [SSH Keys Variable](#ssh-keys-variable)
-    - [Virtual Servers Variable](#virtual-servers-variable)
-6. [Cluster and Worker pool](#cluster-and-worker-pool)
-7. [IBM Cloud Services](#ibm-cloud-services)
-8. [Virtual Private Endpoints](#virtual-private-endpoints)
-9. [IBM Cloud Services](#ibm-cloud-services-1)
-    - [Cloud Object Storage](#cloud-object-storage)
-10. [VPC Placement Groups](#vpc-placement-groups)
-11. [Security and Compliance Center](#security-and-compliance-center)
-12. [Module Variables](#module-variables)
-13. [Contributing](#contributing)
-14. [Terraform Language Resources](#terraform-language-resources)
-15. [Using This Architecure as a Template for Multiple Patterns](#as-a-template-for-multiple-patterns)
-16. [Creating An Issue](#creating-an-issue)
-
----
-
 ## VPC
 
-![vpc-module](../.docs/vpc-module.png)
+![vpc-module](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/main/.docs/vpc-module.png?raw=true)
 
-This template allows users to create any number of VPCs in a single region. The VPC network and components are created by the [Cloud Schematics VPC module](https://github.com/Cloud-Schematics/multizone-vpc-module). VPC components can be found in [main.tf](./main.tf)
+This template allows users to create any number of VPCs in a single region. The VPC network and components are created by the [Cloud Schematics VPC module](https://github.com/Cloud-Schematics/multizone-vpc-module). The VPC components are described in the [main.tf](./main.tf) file.
```

Comment:
Just realizing that this one is not accurate. The VPC module is here: https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc

Reply:
changed link text to SLC module and the link to https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc